appveyor.yml

```diff
@@ -0,0 +1,17 @@
+version: "{build}"
+
+clone_folder: c:\projects\vim-ci
```

Comment:
Oops! This line is not needed. We can use the default folder.
I will update later.